integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/hive/SparkHiveCatalogIT.java

```diff
@@ -201,6 +201,25 @@ void testHiveFormatWithStoredAs() {
     checkParquetFile(tableInfo);
   }
 
+  @Test
+  void testHiveFormatWithExternalTable() {
+    String tableName = "test_hive_format_with_external_table";
+    dropTableIfExists(tableName);
+    String createTableSql = getCreateExternalTableString(tableName);
+    sql(createTableSql);
+    SparkTableInfo tableInfo = getTableInfo(tableName);
+
+    SparkTableInfoChecker checker =
+        SparkTableInfoChecker.create()
+            .withName(tableName)
+            .withTableProperties(
+                ImmutableMap.of(
+                    HivePropertiesConstants.GRAVITINO_HIVE_TABLE_TYPE,
```

Comment:
you should transform Gravitino hive properties to spark properties, and check with spark properties